test/Redistribution.test.ts

```diff
@@ -101,13 +103,15 @@ const depth_4 = '0x06';
 let node_5: string;
 const overlay_5 = '0x676720d79d609ed462fadf6f14eb1bf9ec1a90999dd45a671d79a89c7b5ac9d8';
 const stakeAmount_5 = '100000000000000000';
+const effectiveStakeAmount_5 = '99999999999984000';
```

Comment:
because of this

uint256 _addCommittedStake = _addAmount / OracleContract.currentPrice();
which is 10 n16 /  24000    and some decimals are lost. 

So from the start the committed stake is a bit lower than the potential stake. Was thinking to discuss this and whether should we mitigate this in some way.